static/app/components/group/groupSummary.tsx

```diff
@@ -50,6 +50,29 @@ export const makeGroupSummaryQueryKey = (
   },
 ];
 
+/**
+ * Gets the data for group summary if it exists but doesn't fetch it.
+ */
+export function useGroupSummaryData(
+  group: Group,
+  event: Event | null | undefined,
+  forceEvent = false
+) {
+  const organization = useOrganization();
+  const queryKey = makeGroupSummaryQueryKey(
+    organization.slug,
+    group.id,
+    forceEvent ? event?.id : undefined
```

Comment:
i don't think you need `forceEvent` set to true ever in this use case. That's only needed for forcing it to generate a new summary